legal-api/src/legal_api/models/business.py

```diff
@@ -208,23 +229,34 @@ def delete(self):
             self.save()
         return self
 
+    def get_min_ar_datetime(self):
+        """Get min annual report datetime in UTC time."""
+        ar_min_date = self.get_ar_dates((self.last_ar_year if self.last_ar_year else self.founding_date.year) + 1)[0]
+        ar_min_datetime = datetime(ar_min_date.year, ar_min_date.month, ar_min_date.day)
+        return ar_min_datetime.astimezone(timezone.utc)
+
     def json(self):
         """Return the Business as a json object.
 
         None fields are not included.
         """
+        ar_min_date, ar_max_date = self.get_ar_dates(
+                                    (self.last_ar_year if self.last_ar_year else self.founding_date.year) + 1
+                                   )
         d = {
             'foundingDate': self.founding_date.isoformat(),
             'identifier': self.identifier,
             'lastModified': self.last_modified.isoformat(),
             'lastAnnualReport': datetime.date(self.last_ar_date).isoformat() if self.last_ar_date else '',
-            'nextAnnualReport': self.next_anniversary.isoformat(),
+            'nextAnnualReport': self.get_min_ar_datetime().isoformat(),
```

Comment:
I would suggest to make `get_min_ar_datetime` as a helper function by passing `ar_min_date` as a parameter. You already have `ar_min_date` in `json` function. Try to avoid multiple calls to `get_ar_dates`.

Reply:
Ok! I removed get_ar_min_date function. Thanks!